xmodule/library_content_block.py

```diff
@@ -436,15 +504,11 @@ def author_view(self, context):
                 max_count = self.max_count
                 if max_count < 0:
                     max_count = len(self.children)
-
-                fragment.add_content(self.runtime.service(self, 'mako').render_cms_template(
-                    "library-block-author-preview-header.html", {
-                        'max_count': max_count,
-                        'display_name': self.display_name or self.url_name,
-                    }))
+                context = {} if not context else copy(context)
                 context['can_edit_visibility'] = False
                 context['can_move'] = False
                 context['can_collapse'] = True
+                context['selectable'] = True
                 self.render_children(context, fragment, can_reorder=False, can_add=False)
```

Comment:
It would be awesome to set `can_reorder=True` when `shuffle==False`, but I realize there's some complexity around that, so I'm very OK with leaving `can_reorder=False` for now.

That said: Users can user re-orders the candidates in @rayzhou-bit 's editor, right? If so, is it correct that the order will be saved into candidates, but not children? If yes to both of those, then the order of this view may be wrong. I'm OK with leaving that as an issue for now, but I'd want to flag it for follow-up in the V2 effort.